tests/main/interfaces-ros-snapd-support/task.yaml

```diff
@@ -0,0 +1,51 @@
+summary: Ensure that the ros-snapd-support interface works.
+
+details: |
+    The ros-snapd-support interface allows the ros-snapd snap to
+    control snap applications.
+
+    The ros-snapd and ros2-snapd snaps define the ros-snapd-support plug.
+
+    The snaps declaring a plug on this interface must be able to control
+    snap applications through the snapd-snap.socket.
+    The test snap used has a command to get the list of snap applications
+    (exercising the read capability on the socket) and a command stop a snap application
+    (which checks the write capability).
+
+environment:
+    # not all terminals support UTF-8, but Python tries to be smart and attempts
```

Comment:
XXX: This is not about terminal at all.

Reply:
Done in https://github.com/canonical/snapd/pull/15300/commits/12f63e78042346130082e79ebe9298cd937cc608